content/concepts/loaders.md

```diff
@@ -1,6 +1,163 @@
+
+### Referencing loaders
+
+By convention, though not required, loaders are usually named as `XXX-loader`, where `XXX` is the context name. For example, `json-loader`.
+
+You may reference loaders by its full (actual) name (e.g. `json-loader`), or by its shorthand name (e.g. `json`).
+
+The loader name convention and precedence search order is defined by [`resolveLoader.moduleTemplates`](http://webpack.github.io/docs/configuration.html#resolveloader-moduletemplates) within the webpack configuration API.
+
+Loader name conventions may be useful, especially when referencing them within `require()` statements; see usage below.
+
+### Installing loaders
+
+If the loader is available on npm you can install the loader via:
+
+```sh
+$ npm install xxx-loader --save
+```
+
+or
+
+```sh
+$ npm install xxx-loader --save-dev
+```
+## Usage
+
+There are multiple ways to use loaders in your app:
+
+* explicit in the `require` statement
+* configured via configuration
+* configured via CLI
+
+### loaders in `require`
+
+> **Note:** Avoid using this, if at all possible, if you intend your scripts to be environment agnostic (node.js and browser). Use the *configuration* convention for specifying loaders (see next section).
+
+It's possible to specify the loaders in the `require` statement (or `define`, `require.ensure`, etc.). Just separate loaders from resource with `!`. Each part is resolved relative to the current directory.
+
+It's possible to overwrite any loaders in the configuration by prefixing the entire rule with `!`.
+
+**some-node-script.js**
+
+```javascript
+require("./loader!./dir/file.txt");
+// => uses the file "loader.js" in the current directory to transform
+//    "file.txt" in the folder "dir".
+
+require("jade!./template.jade");
+// => uses the "jade-loader" (that is installed from npm to "node_modules")
+//    to transform the file "template.jade"
+//    If configuration has some transforms bound to the file, they will still be applied.
+
+require("!style!css!less!bootstrap/less/bootstrap.less");
+// => the file "bootstrap.less" in the folder "less" in the "bootstrap"
+//    module (that is installed from github to "node_modules") is
+//    transformed by the "less-loader". The result is transformed by the
+//    "css-loader" and then by the "style-loader".
+//    If configuration has some transforms bound to the file, they will not be applied.
+```
+
+
+### Configuration
+
+You can bind loaders to a RegExp via [configuration](./configuration):
+
+**webpack.config.js**
+
+```javascript
+{
+	module: {
+		rules: [
+			{ test: /\.jade$/, loader: "jade" },
+			// => "jade" loader is used for ".jade" files
+
+			{ test: /\.css$/, loader: "style!css" },
+			// => "style" and "css" loader is used for ".css" files
+			// Alternative syntax:
+			{ test: /\.css$/, use: ["style", "css"] },
+		]
+	}
+}
+```
+
```

Comment:
Newest beta might have some new syntax so better check that out (see how to migrate docs for more info).